build/build.go

```diff
@@ -26,3 +26,13 @@ func Runfile() string {
 	// return converted output value
 	return output
 }
+
+// Native function to call Linux command
+func Native() string {
+	cmd, err := exec.Command("/bin/sh", "/Users/richard/Desktop/Go/jekyll-server-build/build/echo.sh").Output()
```

Comment:
Since `cmd` here receives the output, let's name it `output` to be completely clear. (It's usual to create a command object first, and then execute it, which is why you might have seen the `cmd` variable being used.)

Reply:
That makes sense.